### PR TITLE
23 update standard app api dependency

### PR DESCRIPTION
**Pull request by yoavnash**

### CODE REVIEW

File: examples/rpc_server.py [hunk 1]
```diff
@@ -11,7 +11,9 @@
 def my_message_handler(
     request_message: MessageBrokerRequestModel,
 ) -> MessageBrokerResponseModel:
-    print("Routing to endpoint %r..." % request_message.endpoint)
+    print(
+        f"Routing to endpoint {request_message.endpoint} for method {request_message.method}..."
+    )
```
Comment (csadorf) [inline examples/rpc_server.py:16]: Is this change related to updating the dependencies?

Comment (yoavnash) [reply examples/rpc_server.py:16]: No, it's just to show that the information about the http method has been added (see here: https://github.com/materials-marketplace/standard-app-api/pull/36).
